<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagSearchWidget</class>
 <widget class="QWidget" name="TagSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>タグ検索</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="searchPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSearchCriteria">
         <property name="title">
          <string>検索条件</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditKeyword">
            <property name="placeholderText">
             <string>キーワードを入力...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxMatchType">
            <property name="title">
             <string>一致タイプ</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="radioButtonExact">
               <property name="text">
                <string>完全一致</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonPartial">
               <property name="text">
                <string>部分一致</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelFormat">
            <property name="text">
             <string>フォーマット:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxFormat"/>
          </item>
          <item>
           <widget class="QLabel" name="labelType">
            <property name="text">
             <string>タイプ:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxType"/>
          </item>
          <item>
           <widget class="QLabel" name="labelLanguage">
            <property name="text">
             <string>言語:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxLanguage"/>
          </item>
          <item>
           <widget class="QLabel" name="labelUsageCount">
            <property name="text">
             <string>使用回数:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="usageCountSlider" native="true"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSearch">
            <property name="text">
             <string>検索</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxSavedSearches">
         <property name="title">
          <string>保存された検索</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QComboBox" name="comboBoxSavedSearches"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSaveSearch">
            <property name="text">
             <string>現在の検索を保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidgetResults">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabList">
       <attribute name="title">
        <string>リスト表示</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidgetResults"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStatistics">
       <attribute name="title">
        <string>統計</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QWidget" name="ChartPlaceholderWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
